plugin/evm/block_builder.go

```diff
@@ -52,8 +52,8 @@ type blockBuilder struct {
 	ctx         *snow.Context
 	chainConfig *params.ChainConfig
 
-	chain   *subnetEVM.ETHChain
-	network Network
+	chain    *subnetEVM.ETHChain
+	gossiper Gossiper
```

Comment:
`isSE` seems to be a parallel of `isAP4` in coreth, can we get rid of it and the surrounding code considering it has always been active and there's no reason for anyone to launch without it?